### PR TITLE
Fixing the Markdown syntax in checkout_new_field

### PR DESCRIPTION
**Pull request by rafaelstz**
## Purpose of this pull request

Fix the Markdown syntax in the checkout new field page, as you can see below.

Screen Shot 2019-09-16 at 1 25 17 PM

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_new_field.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/howdoi/checkout/checkout_new_field.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Clear the `generated/code` directory when you run the `setup:di:compile` command
 
 If you completed all the steps described in the previous sections, Magento will generate the interface that includes your custom attribute and you can access your field value.
 
-You can set/get these attributes values by creating an instance of the  `Magento/Quote/Api/Data/AddressInterface.php interface`.
+You can set/get these attributes values by creating an instance of the  `Magento/Quote/Api/Data/AddressInterface.php`  interface.
```
Comment (shrielenee) [inline guides/v2.2/howdoi/checkout/checkout_new_field.md]: Are there two spaces before "interface"? Ensure there is only one.

**Comment by rafaelstz** [comment]
@shrielenee done, thank you for letting me know!

**Comment by shrielenee** [comment]
running tests

**Pull request merged**